### DISCUSSION TITLE
Private and Hide commands

### DISCUSSION THREAD

**Discussion by Nazar1ky** [chr233/ASFEnhance#230] [功能建议 / Ideas]
Hello! Idk if that possible, but will be good to add commands to private/unprivate and hide/unhide games

**Comment by chr233** [comment]
is this operation can be done via browser?

**Comment by Nazar1ky** [comment]
There API I think: https://steamapi.xpaw.me/#IAccountPrivateAppsService/ToggleAppPrivacy

**Comment by chr233** [comment]
added in 2.1.3.0

**Comment by Nazar1ky** [comment]
Nice! Thanks!
